cnf-certification-test/lifecycle/suite.go

```diff
@@ -584,23 +582,25 @@ func TestPodTolerationBypass(env *provider.TestEnvironment) {
 	testhelper.AddTestResultLog("Non-compliant", podsWithRestrictedTolerationsNotDefault, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
 func testStorageRequiredPods(env *provider.TestEnvironment) {
-	oc := clientsholder.GetClientsHolder()
-	var nonCompliantPods []*provider.Pod
-	for _, pod := range env.Pods {
-		for _, podOwner := range pod.ObjectMeta.GetOwnerReferences() {
-			if podOwner.Kind != statefulSet {
-				continue
-			}
-			statefulset, err := oc.K8sClient.AppsV1().StatefulSets(pod.Namespace).Get(context.TODO(), podOwner.Name, apiv1.GetOptions{})
-			if err != nil {
-				tnf.ClaimFilePrintf(err.Error())
+	var persistentVolumesBad []string
+	if env.StorageList[index] == "kubernetes.io/no-provisioner" {
+		tnf.ClaimFilePrintf("The class storage is using local storage")
+		return
+	}
```

Comment:
I'm not sure I understand this section of code, is this just a leftover from debugging?

Reply:
In the begining we check if a storageclass is using local storage, if not we print message and thats it , if yes- we check pod by pod. 
I need to change the code to be like this - 
	if env.StorageList[index] != "kubernetes.io/no-provisioner" {
		tnf.ClaimFilePrintf("The class storage not use in local storage")
		return
	}